.github/workflows/upload_release_asset.yml

```diff
@@ -0,0 +1,36 @@
+# fossa report attribution --debug --format spdx >> myFile
+name: Upload SBOM as Release Asset
+# Ensures that this is only run once, either when pushing
+# directly to main OR when pushing to a PR against main.
+on:
+  release:
+    types:
+      - published
+jobs:
+  generate_SBOM:
+    runs-on: ubuntu-latest
+    env:
+          FOSSA_API_KEY: ${{secrets.FOSSAAPIKEY}}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: Download Fossa binary and install
+        run:  |
```

Comment:
It feels a little strange to be downloading and installing a dependency via curl like this. What are the reasons to do it this way, rather than https://github.com/fossas/fossa-action/ (or some other pre-packaged action)?

Reply:
I'll investigate, I wasn't aware they (Fossa) had an action TBH...

Reply:
@BlueWinds per our conversation, I added comments to the code.  For posterity, I'm stating here that utilizing an action authored by Fossa will not yield our intended results or meet our requirements.